instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import './PostContainer.css'
+import CommentSection from '../CommentSection/CommentSection';
+function PostContainer(props){
+    return(
+        <div>
+            {props.posts.map(post=>(
+                <div key={post.id} className="post">
+                    <div className="post-header"><img src={post.thumbnailUrl} alt="thumbnail" className="thumbnail"></img> {post.username}</div>
```

Comment:
When you find yourself writing multiple lines with props references, it could probably be turned into a separate component. ;)